ippf_upgrade.php

```diff
@@ -163,7 +163,7 @@ function do_visit_form($irow, $encounter, $first)
         }
     }
 
-    echo "<p><font color='green'>Done. Inserted $insert_count visit forms.</font></p>\n";
+    echo "<p class='text-success'>Done. Inserted $insert_count visit forms.</p>\n";
```

Comment:
Is this change bs4 specific? If so, this script doesn't include bs4.

Reply:
@bradymiller @stu01509 Maybe it's time to convert this file to BS4 *hint* *hint* *nudge* *nudge*